package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "4.3.22",
+  "version": "5.0.0",
```

Comment:
No change to `vue-test-utils-helpers` spec on line 77?

Reply:
i changed the package-lock.json directly to fix the +ssh to +https   as recommended by @pwei1018

Reply:
That's not ideal -- what will happen the next time someone does `npm i`?

Reply:
Also, what are your thoughts about running `npm audit fix` or `npm audit fix --force`?

Reply:
> `npm audit fix --force`

OK, don't run this one as it results in libraries that don't build, ie:

```
severin@DESKTOP-2XZPXD3:~/repos/business-filings-ui$ npm run serve

> business-filings-ui@5.0.0 serve
> vue-cli-service serve

 INFO  Starting development server...
 ERROR  ValidationError: Progress Plugin Invalid Options

        options should NOT have additional properties
        options should NOT have additional properties
        options should NOT have additional properties
        options should pass "instanceof" keyword validation
        options should match exactly one schema in oneOf

ValidationError: Progress Plugin Invalid Options

options should NOT have additional properties
options should NOT have additional properties
options should NOT have additional properties
options should pass "instanceof" keyword validation
options should match exactly one schema in oneOf

    at validateOptions (/home/severin/repos/business-filings-ui/node_modules/webpack/node_modules/schema-utils/src/validateOptions.js:32:11)
    at new ProgressPlugin (/home/severin/repos/business-filings-ui/node_modules/webpack/lib/ProgressPlugin.js:62:3)
    at new Progress (/home/severin/repos/business-filings-ui/node_modules/progress-webpack-plugin/index.js:25:21)
    at new progressPlugin (/home/severin/repos/business-filings-ui/node_modules/progress-webpack-plugin/index.js:127:10)
    at /home/severin/repos/business-filings-ui/node_modules/webpack-chain/src/Plugin.js:14:18
    at Object.toConfig (/home/severin/repos/business-filings-ui/node_modules/webpack-chain/src/Plugin.js:78:22)
    at /home/severin/repos/business-filings-ui/node_modules/webpack-chain/src/Config.js:129:63
    at Array.map (<anonymous>)
    at module.exports.toConfig (/home/severin/repos/business-filings-ui/node_modules/webpack-chain/src/Config.js:129:40)
    at Service.resolveWebpackConfig (/home/severin/repos/business-filings-ui/node_modules/@vue/cli-service/lib/Service.js:277:34)
    at PluginAPI.resolveWebpackConfig (/home/severin/repos/business-filings-ui/node_modules/@vue/cli-service/lib/PluginAPI.js:132:25)
    at serve (/home/severin/repos/business-filings-ui/node_modules/@vue/cli-service/lib/commands/serve.js:75:31)
    at Service.run (/home/severin/repos/business-filings-ui/node_modules/@vue/cli-service/lib/Service.js:262:12)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
severin@DESKTOP-2XZPXD3:~/repos/business-filings-ui$ npm ci
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated source-map-url@0.4.0: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated chokidar@2.1.8: Chokidar 2 does not receive security updates since 2019. Upgrade to chokidar 3 with 15x fewer dependencies
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

added 1936 packages, and audited 1937 packages in 13s

122 packages are looking for funding
  run `npm fund` for details

5 high severity vulnerabilities

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
```

Reply:
i ran npm audit fix... this is what it says now.

![image](https://user-images.githubusercontent.com/104083911/165592940-48be722d-02dc-479b-9605-5032c1e794b1.png)

 i also fetched upstream, rebased, and merge main into my branch.

after npm i   (+https DOES NOT CONVERT BACK TO +SSH) in package-lock.json

anything else before i push up.